### PR TITLE
Chore(fix): Update Readme and add section Getting Started.

### PR DESCRIPTION
**Pull request by MaxxReis**
Added Getting Started section;
Added instructions to run the application with Auth0 and non-Auth0;
Fix broken links in readme file.

### CODE REVIEW

File: surveyo/schema_noauth.graphql [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+type Response {
+    id: ID!
+    form: Form!
+    entries: [Entry!]!
+}
+
+type Entry {
+    id: ID!
+    field: Field!
+
```
Comment (MichelDiz) [inline surveyo/schema_noauth.graphql:53]: Why this extra line?

File: surveyo/README.md [hunk 2]
```diff
@@ -32,15 +32,34 @@ Surveyo is a sample app that provides users with a survey tool that they can use
 
 ### Back-end
 
-- [Slash GraphQL](https://dgraph.io/slash-graphql)—a fully managed GraphQL backend service
+- [Slash GraphQL](https://dgraph.io/slash-graphql)—a fully managed GraphQL backend service.
```
Comment (MichelDiz) [inline surveyo/README.md:35]: Let's change Slash GraphQL to Dgraph Cloud or something related. We are not using the name "Slash GraphQL" it was the previous name of our cloud.

File: surveyo/README.md [hunk 3]
```diff
@@ -32,15 +32,34 @@ Surveyo is a sample app that provides users with a survey tool that they can use
...
+1. Copy the entire contents of the `schema_auth.graphql` file located in the root folder of the application.
+2. Open your `Graphql Client`, you can check some GraphQL clients [here](https://dgraph.io/docs/graphql/quick-start/#testing-your-graphql-api) or open a `terminal` and run the following command: `curl -X POST localhost:8080/admin/schema --data-binary <@SCHEMA.GRAPHQL>`, which contains the code copied in step 1.
+3. Create an authentication service account on: [Auth0](https://auth0.com/).
+4. After creating your Auth0 account, follow the instructions provided [here](https://dgraph.io/docs/learn/developer/todo-app-tutorial/todo-auth0-jwt/) to set up Auth0 authentication. 
+5. In the root folder of the application, open a terminal and run the following commands: `npm install` followed by `npm start`.
+
 ### Links
 
-- [Deploy Now](https://slash.dgraph.io/_/one-click?app=surveyo)
+<!-- - [Deploy Now](https://slash.dgraph.io/_/one-click?app=surveyo) -->
```
Comment (MichelDiz) [inline surveyo/README.md:58]: Remove this comment. No need to left any references to Slash.

**Review by MichelDiz — CHANGES_REQUESTED** [review]

File: surveyo/schema_noauth.graphql (hunk 1 above)
Comment (MaxxReis) [reply surveyo/schema_noauth.graphql:53]: Done.

File: surveyo/README.md (hunk 2 above)
Comment (MaxxReis) [reply surveyo/README.md]: Done.

File: surveyo/README.md (hunk 3 above)
Comment (MaxxReis) [reply surveyo/README.md]: Done.

File: surveyo/schema_noauth.graphql [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+type Entry {
+    id: ID!
+    field: Field!
+    
+    date: DateTime
+    netPromoterScore: Int
+    rating: Int
+    singleChoice: FieldOption
+    text: String
+}
```
Comment (MichelDiz) [inline surveyo/schema_noauth.graphql:59]: After the line 59 add an extra line. It is EOF, all files needs an extra empty line in the end.

**Review by MichelDiz — APPROVED** [review]

**Pull request merged**